pages/collections/_type/index.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <b-container>
+    <ContentHeader
+      :title="$tc('entity.index.subjects', 2)"
+    />
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          :data-qa="`${ this.$route.params.type} listing page`"
+        >
+          <ContentCard
+            v-for="entity in entities"
+            :key="entity.id"
+            :title="entity.prefLabel[$i18n.locale]"
+            :url="entity.id"
```

Comment:
This results in an external URL to 'data.europeana.eu/...', these then redirect to the production portal collections page.

See how the components/entity/entityCards component converts 'data.europeana.eu/...' enitty URIs to portal URLs by means of a helper method `entityRoute`. However that component shows the cards in the wrong style, so just port the logic from there into the page here.

Alternatively, to avoid this page template from getting too big make a new component 'entityListingCards' which then combines the style from here as well as the logic from the entityCards component to make sure the links go to collection pages and are styled correctly.

Reply:
I've copied the logic from entityCards, it doesn't make the component to big, think it's a better approach than making a new entityListingCards component

Reply:
makes sense.